tests/framework/CodeInterfaceTests/RAVEN/optimizer.xml

```diff
@@ -0,0 +1,145 @@
+<Simulation verbosity="debug">
+  <TestInfo>
+    <name>framework/CodeInterfaceTests/RAVEN.Optimizer</name>
+    <author>talbpaul</author>
+    <created>2018-01-18</created>
+    <classesTested>Models.Code.RAVEN</classesTested>
+    <description>
+      Basic test of the raven-runs-raven interface.
+      Outer raven runs a grid sampling by changing the upper and lower bounds of the sampledVars distribution on the inner raven.
+      Inner raven runs 100 MC samples, does basic stats, and returns the mean, sigma to the outer raven.
+    </description>
+    <analytic>
+      Each sample in this test converges towards the analytic solutions discussed in the "changing lower, upper bounds" section of
+      the "attenuate" analytic model documentation.
+    </analytic>
+    <revisions>
+      <revision author="talbpaul" date="2018-05-02">added "fully-correlated" variable for innerUpperBound as a test of the mechanic</revision>
+      <revision author="talbpaul" date="2018-11-06">modified inner run to use grid for speed, consistent results</revision>
+    </revisions>
```

Comment:
I don't think these revisions apply to this new test.

Reply:
Agreed.